docs/Arbitrary_multi-table_SQL.md

```diff
@@ -0,0 +1,41 @@
+
+## Arbitrary SQL template execution
+
+You can run arbitrary SQL operations on multiple tables. In order for this to work, the dataset of each SQL command should return the same schema. You can align the schemas by:
+* Running schema-agnostic commands (Eg. Vacuum, Optimize, History, Describe, etc.)
+* Specifying the common column names to be selected
+* Melting multiple columns into a "long" format
```

Comment:
Could we provide an example of this?

Reply:
Hmm, maybe we replace this with the to_json example. We have a dedicated function for the string melt.

Reply:
yeah that would make sense